lib/auth/auth_chalresp.c

```diff
@@ -100,32 +101,67 @@ struct auth_chalresp_result {
 #pragma pack(pop)
 
 
+
 /**
- * Shared key.
- * @brief  In a production system, the shared key should be stored in a
- * secure hardware store such as an ECC608A or TrustZone.
+ * Challenge-Response instance info.  Used to set shared key for each
+ * auth instance.
  */
-static uint8_t default_shared_key[AUTH_SHARED_KEY_LEN] = {
```

Comment:
Removed default shared key out of fear users of this library would just rely on this key.